libraries/Microsoft.Bot.Builder.Dialogs/Constants/DialogPath.cs

```diff
@@ -1,6 +1,37 @@
-﻿namespace Microsoft.Bot.Builder.Dialogs
+﻿using System.Data;
+
+namespace Microsoft.Bot.Builder.Dialogs
 {
     public class DialogPath
     {
+        /// <summary>
+        /// Counter of emitted events.
+        /// </summary>
+        public const string EventCounter = "dialog.eventCounter";
+
+        /// <summary>
+        /// Information for tracking memory changes.
+        /// </summary>
+        public const string Tracker = "dialog.tracker";
+
+        /// <summary>
+        /// Information for tracking when trigger handlers last ran.
+        /// </summary>
+        public const string ConditionTracker = Tracker + ".conditions";
+
+        /// <summary>
+        /// Currently expected properties.
+        /// </summary>
+        public const string ExpectedProperties = "dialog.expectedProperties";
+
+        /// <summary>
+        /// Last surfaced event from a form dialog.
+        /// </summary>
+        public const string LastEvent = "dialog.LastEvent";
```

Comment:
>LastEvent [](start = 48, length = 9)

the case of this doesn't match the other constants

Reply:
Memory is actually case insensitive, but I will change it to be consistent.

---
In reply to: [343889805](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r343889805) [](ancestors = 343889805)